templates/desktop/organisations.html

```diff
@@ -36,116 +36,78 @@ <h1>Bring Ubuntu to your <br> organisation</h1>
-        <p><a href="/security">Learn more about Ubuntu's security practices&nbsp;&rsaquo;</a></p>
-      </div>
+    <div class="col-4 p-card col-medium-3">
+      <h4>Compliance, certification and hardening</h4>
+      <hr />
+      <p>
+        Ubuntu Pro delivers security and compliance for even the most sensitive workloads with <a href="/security/certifications#fips">FIPS 140-2 certified modules</a>, <a href="/security/certifications#cis">CIS hardening</a> and <a href="/security/certifications#common-criteria">Common Criteria EAL2</a> certification available.
+      </p>
+      <p>
+        <a href="/security">
+          Learn more about Ubuntu's security practices
+        </a>
+      </p>
     </div>
-    <div class="col-4">
-      <div class="u-hide--small u-hide--medium">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/7cafcb2c-Managed-apps.svg",
-          alt="",
-          width="197",
-          height="100",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </div>
-      <h3>Certified hardware</h3>
-      <div>
-        <p>Ubuntu Desktop is certified across a wide range of laptops, desktops, and workstations, including the HP Z series and Dell  XPS and Precision lines.</p>
-        <p><a href="/certified">Learn more about Ubuntu certified hardware&nbsp;&rsaquo;</a></p>
-      </div>
+    <div class="col-4 p-card col-medium-3">
+      <h4>Certified hardware</h4>
+      <hr />
+      <p>
+        Ubuntu Desktop is certified across a wide range of laptops, desktops, and workstations, including the HP Z series and Dell  XPS and Precision lines.
+      </p>
+      <p>
+        <a href="/certified">
+          Learn more about Ubuntu certified hardware
+        </a>
+      </p>
     </div>
-    <div class="col-4">
-      <div class="u-hide--small u-hide--medium">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/e9d3cf66-We+support+your+operations.svg",
-          alt="",
-          width="100",
-          height="100",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </div>
-      <h3>Weekday or 24/7 Support</h3>
-      <div>
-        <p>UAdvanced support customers get direct 24/7 access to our enterprise open source support team through our web portal, knowledge base or via phone.</p>
-        <p><a href="/desktop/contact-us" class="js-invoke-modal p-button">Contact us</a></p>
-      </div>
+    <div class="col-4 p-card col-medium-3">
+      <h4>Weekday or 24/7 Support</h4>
+      <hr />
+      <p>
+        UAdvanced support customers get direct 24/7 access to our enterprise open source support team through our web portal, knowledge base or via phone.
```

Comment:
Can you remove the "U"? There is no U in the [copy doc](https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#)